svo (namedb).xml

```diff
@@ -3235,6 +3235,10 @@ function ndb.downloaded_file(_, filename)
     if city ~= "(hidden)" then
         temp_name_list[1].city = (city ~= "(none)" and string.title(city) or "rogue")
     end
+    -- API returns 'underworld' for the Undead.
+    if city == 'underworld' then
```

Comment:
This will overwrite the city value twice (once for the if clause above, once here). It's fine in this case, I just wanted to point it out as a learning opportunity :wink: